modules/kubernetes/aks-core/variables.tf

```diff
@@ -647,10 +616,12 @@ variable "coredns_upstream" {
   default     = false
 }
 
-variable "use_private_ingress" {
```

Comment:
Why is this being removed?

Reply:
Acording to the linter, the variable isn't used.